<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineProperties</class>
 <widget class="QDialog" name="LineProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Initial Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Network:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="network"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Initial Bar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="noI"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Final Bar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="noF"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="length"/>
        </item>
        <item>
         <widget class="QLabel" name="lengthUn">
          <property name="text">
           <string>[m]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Impedance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Zaa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaa"/>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaai"/>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn1">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Zab</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zab"/>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zabi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn2">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Zac</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zac"/>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zaci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn3">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Zbb</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbb"/>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbbi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_46">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn5">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Zbc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbc"/>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zbci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn6">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Zcc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcc"/>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Zcci"/>
        </item>
        <item>
         <widget class="QLabel" name="label_49">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="impedanceUn8">
          <property name="text">
           <string>[ohm]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
